docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+```
+{ .namespace = "prod" }
+```
+
+Find any database connection string that goes to a Postgres or MySQL database:
+```
+{ .db.system =~ "postgresql|mysql" }
+```
+
+### Scoped attribute fields
+
+Attributes can be specifically scoped to either "span" or "resource". Specifying "span" or "resource" can result in significant performance benefits because it reduces the amount of data that Tempo has to retrieve to answer your query.
+
+For example, to find traces with a span attribute of `http.status` set to `200`:
+```
+{ span.http.status = 200 }
+```
+
+To find traces where a the resource `namespace` is set to `prod`:
+```
+{ resource.namespace = "prod" }
+```
+
+<p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span resources and attributes" /></p>
+
+
+### Comparison operators
+
+Comparison operators are used to test values within an expression.
+
+The implemented comparison operators are:
+
+- `=` (equality)
+- `!=` (inequality)
+- `>` (greater than)
+- `>=` (greater than or equal to)
+- `<` (less than)
+- `<=` (less than or equal to)
+- `=~` (regular expression)
+
+For example, to find all traces where an `http.status` attribute in a span are greater than `400` but less than equal to `500`:
+
+```
+{ .http.status >= 400 && .http.status < 500 }
+```
+
+Find all traces where the `http.method` attribute is either `GET` or `DELETE`:
+
+```
+{ .http.method =~ “DELETE|GET” }
+```
+
+### Field expressions
+
+Fields can also be combined in various ways to allow more flexible search criteria. A field expression is a composite of multiple fields that define all of the criteria that must be matched to return results.
+
+#### Examples
+
+Find traces with "success" http status codes:
+
+```
+{ .http.status >= 200 && .http.status < 300 }
+```
+
+Find traces where a `DELETE` HTTP method was used and the instrinsic span status was not OK:
+
+```
+{ .http.method = "DELETE" && status != ok }
+```
+
+Both expressions require all conditions to be true on the same span. The entire expression inside of a pair of `{}` must be evaluated as true on a single span for it to be included in the result set.
+
+In the above example, if a span includes an `.http.method` attribute set to `DELETE` where the span also includes a `status` attribute set to `ok`, the trace would not be included in the returned results.
+
+## Combining spansets
+
+Spanset operators let you combine two sets of spans using and (`&&`) as well as union (`||`).
+
+- `{condA} && {condB}`
+- `{condA} || {condB}`
+
+
+For example, if you want to find a trace that went through two specific regions:
+
+```
+{ .region = "eu-west-0" } && { .region = "eu-west-1" }
+```
+
+Note the difference between the above and the following:
+
+```
+{ .region = "eu-west-0" && .region = "eu-west-1" }
+```
+
+The second expression returns no traces because it's impossible for a single span to have a `.region` attribute that is set to both region values at the same time.
```

Comment:
```suggestion
The second expression returns no traces because it's impossible for a single span to have a  `resource.region` attribute that is set to both region values at the same time.
```